### DISCUSSION TITLE
How do I pull a subset of files, specifically anything that isn't a .mp4?

### DISCUSSION THREAD

**Discussion by AngCamp** [dandi/helpdesk#118] [General]
I want to pull this dataset: https://dandiarchive.org/dandiset/000409/draft  I have a conda environment set up for the dandi package.  Is there a way to get `dandi download  http://some_dandiset` to pull just the files I want?  I am not interested in the mp4 files of the raw video footage.  I'm more interested in the neuropixels ephys and behavioral data anyway.

**Comment by satra** [comment]
you can do something like this:

```shell
dandi download 'https://dandiarchive.org/api/dandisets/000409/versions/draft/assets/?glob=sub-CSHL049/*behavior.nwb'
```

also you can use datalad which allows you to create a shell of a dandiset in your local system. 
```shell
datalad install -0
cd 000409/sub-CSHL049
datalad get sub-CSHL049_ses-c99d53e6-c317-4c53-99ba-070b26673ac4-processed-only_behavior.nwb
```

you can also remove the bytes of the file locally:

```
datalad drop sub-CSHL049_ses-c99d53e6-c317-4c53-99ba-070b26673ac4-processed-only_behavior.nwb
```

finally if you are programming you can access each file remotely without downloading all of it.
